products/BellHybrid/apps/application-bell-onboarding/windows/OnBoardingShortcutsOptionWindow.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <common/options/BellShortOptionWindow.hpp>
+#include <apps-common/ApplicationCommon.hpp>
+
+namespace gui
+{
+    class OnBoardingShortcutsOptionWindow : public BellShortOptionWindow
+    {
+      public:
+        explicit OnBoardingShortcutsOptionWindow(app::ApplicationCommon *app, const std::string &name);
+
+      private:
+        app::ApplicationCommon *app;
+        std::list<Option> settingsOptionsList();
```

Comment:
Could this function be `const`?

Reply:
It cannot be `const` because invalid conversion `gui::AppWindow*`.